services/order.py

```diff
@@ -0,0 +1,36 @@
+from db.models import Order, Ticket, MovieSession, User
+from django.db import transaction
+from typing import Optional
+
+
+def create_order(
+    tickets: list[dict],
+    username: str,
+    date: Optional[str] = None,
+) -> Order:
+
+    with transaction.atomic():
+        user = User.objects.get(username=username)
+        order = Order.objects.create(user=user)
+        if date:
+            order.created_at = date
+        order.save()
+
+        for ticket in tickets:
+            movie_session = MovieSession.objects.get(
+                id=ticket["movie_session"]
+            )
+            ticket = Ticket(
+                row=ticket["row"],
+                seat=ticket["seat"],
+                movie_session=movie_session,
+                order=order)
+            ticket.save()
+        return order
+
+
+def get_orders(username: str = None) -> Order:

```

Comment:
use typehint like: Optional[YOUR_VALUE_TYPE] = None
for example: list that contains string elements and can be None has typehint Optional[list[str]]